server/db/models/Tab.js

```diff
@@ -0,0 +1,55 @@
+const { stateTaxTable } = require('./stateTax');
+const db = require('../db');
+const {
+  Model,
+  DataTypes: { ENUM, FLOAT },
+} = require('sequelize');
+const {
+  models: { Order },
+} = db;
+
+class Tab extends Model {
+  getTax() {
+    const venue = Venue.findAll({ where: { venueId: id } });
```

Comment:
- 'venueId' should be 'id'. Also, where is id coming from?
- this can probably be shortened to Venue.findOne(id);

Reply:
Thanks Arjan, I see what you mean. So id is tab.venueId. So i'm trying to pull the entire venue instance of the venue associated to the tab so i can grab venue.state (to calculate the tax). does that make sense?